Competition2017/src/competition/subsystems/drive/commands/DriveToOrientationCommand.java

```diff
@@ -0,0 +1,60 @@
+package competition.subsystems.drive.commands;
+
+import com.google.inject.Inject;
+import xbot.common.command.BaseCommand;
+import xbot.common.math.ContiguousHeading;
+import xbot.common.math.PIDManager;
+import xbot.common.properties.XPropertyManager;
+import competition.subsystems.drive.DriveSubsystem;
+
+public class DriveToOrientationCommand extends BaseCommand{
+
+    DriveSubsystem drive;
+    ContiguousHeading targetHeading;
+    ContiguousHeading currentHeading;
+    
+    public final double defaultPValue = 1/80d;
+    double headingError;
+    
+    private XPropertyManager propMan;
+    private PIDManager headingDrivePid;
+    
+    @Inject
+    public DriveToOrientationCommand(DriveSubsystem driveSubsystem, XPropertyManager propMan) {
+
+        this.drive = driveSubsystem;
+        this.propMan = propMan;
+        
+        headingDrivePid = new PIDManager("HeadingModule", propMan, defaultPValue, 0, 0);
```

Comment:
And based on the command rename, it could be something like "Rotate to heading", which would generate "Rotate to heading P", "Rotate to heading I", etc...